CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Support for HTTP/2 (server-side) ([#3847](https://github.com/opensearch-project/OpenSearch/pull/3847))
 - Add getter for path field in NestedQueryBuilder ([#4636](https://github.com/opensearch-project/OpenSearch/pull/4636))
 - Allow mmap to use new JDK-19 preview APIs in Apache Lucene 9.4+ ([#5151](https://github.com/opensearch-project/OpenSearch/pull/5151))
+- Events Correlation Engine ([#7132](https://github.com/opensearch-project/OpenSearch/pull/7132))
```

Comment:
Please use the imperative mood here, e.g.: "Add events correlation engine plugin". I also think it is better to link to the meta issue instead of just this PR (https://github.com/opensearch-project/OpenSearch/issues/6854). This entry should be in `[Unreleased 2.x]` as well. Also, we'll only want one changelog entry for this issue, so you can include the entry in this PR or the final one, but the key is that the incremental PRs don't need separate entries since all these PRs will be in service of introducing a single new feature/plugin.

Reply:
made changes accrodingly